### PR TITLE
Primero Articulo

### PR DESCRIPTION
**Pull request by uncle-jj**
de prueba

### CODE REVIEW

File: _posts/es/newsletters/2023-02-15-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
+---
+title: 'Boletín Bitcoin Optech #238'
+permalink: /en/newsletters/2023/02/15/
```
Comment (1bitaldia) [inline _posts/es/newsletters/2023-02-15-newsletter.md]: change _en_ to _es_

File: _posts/es/newsletters/2023-02-15-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+name: 2023-02-15-newsletter
```
Comment (1bitaldia) [inline _posts/es/newsletters/2023-02-15-newsletter.md:4]: should append -es

File: _posts/es/newsletters/2023-02-15-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+slug: 2023-02-15-newsletter
```
Comment (1bitaldia) [inline _posts/es/newsletters/2023-02-15-newsletter.md:5]: append -es here also

File: _posts/es/newsletters/2023-02-15-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+type: newsletter
+layout: newsletter
+lang: en
```
Comment (1bitaldia) [inline _posts/es/newsletters/2023-02-15-newsletter.md]: change to es

**Review by kumulydev — CHANGES_REQUESTED** [review]
Check contribution guidelines: https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md#translations

This review is based in contribution guidelines rules, not how spanish is written or the steps to check if the newsletter looks ok in your local website.

you should also check your commit name based on the guidelines

Really happy to see this getting shape 💯

**Pull request closed without merging**